pandas/tests/series/methods/test_convert_dtypes.py

```diff
@@ -298,6 +299,19 @@ def test_convert_dtypes_pyarrow_null(self):
         expected = pd.Series([None, None], dtype=pd.ArrowDtype(pa.null()))
         tm.assert_series_equal(result, expected)
 
+    @td.skip_if_no("pyarrow")
+    def test_convert_empty_categorical_to_pyarrow(self):
+        # GH#59934
+        ser1 = pd.Series(pd.Categorical([None] * 5))
+        converted1 = ser1.convert_dtypes(dtype_backend="pyarrow")
+        expected = ser1
+        tm.assert_series_equal(converted1, expected)
+
+        ser2 = pd.Series(pd.Categorical([None] * 5, categories=["S1", "S2"]))
```

Comment:
Could you use `@pytest.mark.parameterize("categories", [None, ["S1", S2"])` to deduplicate this test?